GameData/RP-1/Contracts/Lunar Crewed/CrewedMoonLandingTargeted.cfg

```diff
@@ -221,14 +221,15 @@ CONTRACT_TYPE
 			{
 				name = TargetedLanding
 				type = Any
+				completeInSequence = true
```

Comment:
I don't get the addition of this. Doesn't make any sense to me.

Reply:
It's to seal a possible loophole that, you fly past the target area at low altitude, and then land at somewhere else